cg_hermes/config/balsamic_qc.py

```diff
@@ -0,0 +1,46 @@
+"""Tags that are defined in Balsamic deliverables mapped to tags used in CG
+
+The tag sets that exists in all files are set to mandatory. Tag sets that exists in < 4 deliverables are not
+mandatory by default. However the tags that are available to a particular analysis is mandatory for that analysis.
+"""
+
+from cg_hermes.config.balsamic import QC_TAGS, RAW_TAGS
+
+
+QC_ALIGNMENT_TAGS = {
+    # Alignment files (PANEL & WGS)
+    frozenset(RAW_TAGS["tumor.merged.cram"]): {  # cram (tumor)
+        "tags": ["tumor", "qc-cram"],
+        "is_mandatory": True,
+        "used_by": ["deliver", "scout"],
```

Comment:
For balsamic-qc analysis, does it makes sense to upload only cram files to Scout? We don't have VCFs for this analysis. may be only caesar delivery isn't good enough??